javascript/organizing_your_javascript_code/objects_and_object_constructors.md

```diff
@@ -150,10 +151,7 @@ console.log(theHobbit.info());
 
 Before we go much further, there's something important you need to understand about JavaScript objects. All objects in JavaScript have a `prototype`. Stated simply, the prototype is another object that the original object _inherits_ from, which is to say, the original object has access to all of its prototype's methods and properties.
 
-This concept is an important one, so you've got some reading to do. Make sure you really get this before moving on!
-
-1. [This article](https://web.archive.org/web/20200513181548/https://javascriptissexy.com/javascript-prototype-in-plain-detailed-language/) is a straightforward introduction and demonstration of the concept. It also covers constructors again.. good time for a review! The important bits here, once you've covered the basics, are 'Prototype-based inheritance' and the 'Prototype chain'.
-2. To go a bit deeper into both the chain and inheritance, spend some time with [this great article](http://javascript.info/prototype-inheritance). As usual, doing the exercises at the end will help cement this knowledge in your mind. Don't skip them! Important note: This article makes heavy use of `__proto__` which is not generally recommended. The concepts here are what we're looking for at the moment. We will soon learn another method or two for setting the prototype.
+The concept of the prototype is an important one, so you’ve got some reading to do, which you'll find in the Assignment section below. Make sure you really get it before moving on!
```

Comment:
I feel like the extra work is a non-issue. We are moving towards having our lessons have all the required readings in the assignment section, rather than sprinkled throughout the lesson.

If it helps, we also have a lesson navigation tool on every lesson page:
![image](https://user-images.githubusercontent.com/85733202/171996399-3bd8bd3c-bf49-4b39-84c1-62e03541cb7d.png)